manifest.json

```diff
@@ -0,0 +1,144 @@
+      "files": "d3.min.js",
+      "jsFiles": ["directories.THIRD_PARTY_DIR/d3js-3.4.11/d3.min.js"]
+    },
+    "ngInfiniteScroll": {
+      "version": "1.0.0",
+      "path": "https://raw.github.com/BinaryMuse/ngInfiniteScroll/1.0.0/build/",
+      "dest_target": "nginfinitescroll-",
+      "files": ["ng-infinite-scroll.min.js"],
+      "jsFiles": ["ng-infinite-scroll.min.js"]
+    },
+    "yuiCompressor": {
+      "version": "2.4.8",
+      "path": "https://github.com/yui/yuicompressor/releases/download/v2.4.8",
+      "dest_target": "yuicompressor-",
+      "dest_jar": "yuicompressor-2.4.8.jar"
+    },
+    "ui-bootstrap": {
+      "version": "0.13.4",
+      "path": "https://raw.githubusercontent.com/angular-ui/bootstrap/gh-pages",
+      "dest_target": "ui-bootstrap-",
+      "files": ["ui-bootstrap-tpls-0.13.4.js", "ui-bootstrap-tpls-0.13.4.min.js"],
+      "jsFiles": ["ui-bootstrap-tpls-0.13.4.min.js"]
+    },
+    "jquery": {
+      "version": "2.1.1",
+      "path": "https://ajax.googleapis.com/ajax/libs/jquery/2.1.1",
+      "dest_target": "jquery-",
+      "files": ["jquery.js", "jquery.min.js", "jquery.min.js.map"],
+      "jsFiles": ["jquery.js"]
+    },
+    "jqueryUI": {
+      "version": "1.10.3",
+      "path": "https://ajax.googleapis.com/ajax/libs/jqueryui/1.10.3",
+      "dest_target": "jquery-",
+      "files": ["jquery-ui.min.js"],
+      "jsFiles": ["jquery-ui.min.js"]
+    },
+    "bowerMaterial": {
+      "version": "0.6.0-rc1",
+      "zip_path": "https://github.com/angular/bower-material/archive/v0.6.0-rc1.zip",
+      "dest_target": "bower-material-",
+      "jsFiles": "",
+      "cssFiles": ["angular-material.min.css"]
+    },
+    "fontAwesome": {
+      "version": "4.4.0",
+      "zip_path": "https://github.com/FortAwesome/Font-Awesome/archive/v4.4.0.zip",
+      "dest_target": "font-awesome-",
+      "fonts_path": "fonts",
+      "cssFiles": ["css/font-awesome.min.css"]
+    },
+    "textAngular": {
+      "version": "1.3.7",
+      "zip_path": "https://github.com/fraywing/textAngular/archive/v1.3.7.zip",
+      "dest_target": "textAngular-",
+      "jsFiles": "",
+      "cssFiles": ["src/textAngular.css"]
+    },
+    "uiSortable": {
+      "version": "0.12.6",
+      "zip_path": "https://github.com/angular-ui/ui-sortable/archive/src0.12.6.zip",
+      "dest_root": "ui-sortable-src",
+      "dest_target": "ui-sortable-",
+      "jsFiles": ""
+    },
+    "ngJoyride": {
+      "version": "0.1.11",
+      "zip_path": "https://github.com/abhikmitra/ng-joyride/archive/0.1.11.zip",
+      "dest_target": "ng-joyride-",
+      "jsFiles": "",
+      "cssFiles": ["ng-joyride.css"]
+    },
+    "bootstrap": {
+      "version": "3.3.4",
+      "zip_path": "https://github.com/twbs/bootstrap/releases/download/v3.3.4/3.3.4.zip",
+      "dest_root": "bootstrap-3.3.4-dist",
+      "dest_target": "bootstrap-",
+      "jsFiles": "",
+      "fonts_path": "fonts",
+      "cssFiles": ["css/bootstrap.min.css"]
```

Comment:
Sure this is unordered but node will parse ```manifest.json```  from start of the file to end of the file and when collecting js files here bootstrap files will be loaded after ngJoyride files.

yap jsFile should be a  list.